xarray/core/variable.py

```diff
@@ -1541,15 +1542,16 @@ def stack(self, dimensions=None, **dimensions_kwargs):
             result = result._stack_once(dims, new_dim)
         return result
 
-    def _unstack_once_full(self, dims: Mapping[Any, int], old_dim: Hashable) -> Self:
+    @deprecate_dims
+    def _unstack_once_full(self, dim: Mapping[Any, int], old_dim: Hashable) -> Self:
```

Comment:
same here: this seems like a internal function, so it should be fine to change without warning